dracut/30ignition/ignition-generator

```diff
@@ -74,9 +74,9 @@ if [ -n "$RANDOMIZE_DISK_GUID" ]; then
     add_requires "disk-uuid@${escaped_guid}.service"
 fi
 
-oem_id=pxe
+detected_platform_id=pxe
 if [[ $(systemd-detect-virt || true) =~ ^(kvm|qemu)$ ]]; then
-    oem_id=qemu
+    detected_platform_id=qemu
```

Comment:
On second thought, I think this block should be deleted.  In CL the qemu images don't explicitly embed an OEM ID, so we have this code as a fallback.  But we should really just be explicit everywhere, with the QEMU images knowing they're QEMU images.  As for PXE, I think that should be a separate attribute now, rather than a platform; for example someone might PXE-boot VMware and want to run its agent.